scripts/apothecary/apothecary

```diff
@@ -88,7 +88,8 @@ VS_64_BIT_ENV='VC\bin\x86_amd64\vcvarsx86_amd64.bat'
 # paths to android SDK, etc
```

Comment:
the latest apothecary is in it's own repository now, in https://github.com/openframeworks/apothecary  can you send this changes there?